genesis/src/main.rs

```diff
@@ -471,6 +479,20 @@ fn main() -> Result<(), Box<dyn error::Error>> {
 
     let cluster_type = cluster_type_of(&matches, "cluster_type").unwrap();
 
+    // Get the features to disable if provided
+    let deactivate_feature_set: HashSet<Pubkey> = matches
+        .values_of("deactivate_feature_set")
+        .map_or(HashSet::new(), |values| {
+            values
+                .map(|s| Pubkey::from_str(s).expect("Invalid Pubkey"))
+                .collect()
+        });
```

Comment:
Think this can be simplified, see:
https://github.com/anza-xyz/agave/blob/923355cbfd13c0bf0ca26db93a2649086e2ef7b9/validator/src/bin/solana-test-validator.rs#L354